app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
+import { Pane, Button, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import {
+  ROLES,
+  CONTACT_FORM_VALIDATION_SCHEMA,
+  CONTACT_FORM_INITIAL_VALUES,
+} from "../constants";
+
+const ContactCreationForm = ({ onClose, isEdit }) => {
```

Comment:
You component naming is wrong. Read: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react